.travis.yml

```diff
@@ -79,7 +79,7 @@ before_install:
   - export PERL5LIB=/usr/share/perl5
 
     # Provide cpanm helper
-  - eval $(curl https://travis-perl.github.io/init) --auto
+  - eval $(curl https://travis-perl.github.io/init | grep -v '#') --auto
```

Comment:
maybe you could update the comment above the line in so when we'll see that in the future we know what we are doing

Reply:
Good idea.
I was also wondering if we should do a better comment removal. I mean what if the comment is at the end of a line?
```
$( curl ... | sed 's/#.+//' )
```

Reply:
I guess there is good chance that the comments are completely removed, but else it would be safer to just remove the comment, and not the entire line. On the other hand, just removing by "#" is never completely safe. It could be part of a command.